src/views/pages/Start/index.jsx

```diff
@@ -107,7 +108,7 @@ const Start = () => {
             onNext={nextStep} />
         }
         {step === STEPS.SHARE_PANEL &&
-          <SharePanel />
+          <SharePanel isSubscribe={0} />
         }
```

Comment:
I'm thinking maybe it would be simpler to rename the `SharePanel` into `SubscribePanel` (it's my bad :p), and then add a new step here for your `SharePage` (which should probably then be renamed into `SharePanel`. What do you think? This way we avoid having the `STEP.XXX` logic inside the Subscribe component, keeping it simpler.

Reply:
As you rightly pointed out there is a dependency between the Subscription and Share panels so my above comment is probably not the best way to handle it